packages/babel-helpers/src/helpers/applyDecs.js

```diff
@@ -306,7 +317,7 @@ function applyMemberDec(
       if (kind === 0 /* FIELD */) {
         initializer = newValue;
       } else if (kind === 1 /* ACCESSOR */) {
-        initializer = newValue.initializer;
+        initializer = newValue.initialize;
```

Comment:
Since we are renaming a property here, I wonder if it makes sense to also support the old one with a warning:
```js
if ((initializer = newValue.initialize) == null && (initializer = newValue.initializer) && typeof console !== "undefined") {
  console.warn(".initialize has been renamed to .init as of March 2022")
}
```

@pzuraq This idea mostly came up while I was reviewing https://github.com/babel/babel/pull/14353, since it's not just a "we implemented the spec wrongly" anymore but a "the proposal had a change that made all the accessor initializers usages invalid".

Reply:
that seems reasonable to me, will update

Reply:
Also, feel free to review/approve this PR since you are the "expert in the field" for decorators! However, I'd like to wait merging this until #14353 is ready so that they can go in the same release.